src/iface-modbus.c

```diff
@@ -182,14 +293,9 @@ int modbus_start(const char *url, int slave_id,
 	return 0;
 }
 
-void modbus_stop(void)
+void iface_modbus_stop(void)
 {
-	if (likely(connect_to))
-		l_timeout_remove(connect_to);
-
-	if (modbus_io)
-		l_io_destroy(modbus_io);
-
-	if (modbus_ctx)
-		connection_interface.destroy(modbus_ctx);
+	l_timeout_remove(connect_to);
+	l_io_destroy(modbus_io);
+	destroy(modbus_ctx);
```

Comment:
I think the name of this function could be more specific as destroy_context or destroy_modbus_ctx or destroy_modbus_context.

Reply:
done